src/main/java/org/assertj/core/api/AbstractDoubleArrayAssert.java

```diff
@@ -807,6 +983,31 @@ public SELF endsWith(double... sequence) {
     return myself;
   }
 
+  /**
+   * Verifies that the actual array ends with the given sequence of values, without any other values between them.
+   * Similar to <code>{@link #containsSequence(Double[])}</code>, but it also verifies that the last element in the
+   * sequence is also last element of the actual array.
+   * <p>
+   * Example:
+   * <pre><code class='java'> // assertion will pass
+   * assertThat(new double[] { 1.0, 2.0, 2.0, 1.0 }).endsWith( new Double[] {2.0, 1.0} );
+   *
+   * // assertion will fail
+   * assertThat(new double[] { 1.0, 2.0, 2.0, 1.0 }).endsWith(new Double[] {1.0, 2.0});</code></pre>
+   *
+   * @param sequence the sequence of values to look for.
+   * @return myself assertion object.
+   * @throws NullPointerException if the given argument is {@code null}.
+   * @throws IllegalArgumentException if the given argument is an empty array.
+   * @throws AssertionError if the actual array is {@code null}.
+   * @throws AssertionError if the actual array does not end with the given sequence.
+   */
+  public SELF endsWith(Double[] sequence) {
```

Comment:
Please also add
```java
public SELF endsWith(Double[] sequence, Offset<Double> precision)
```

Reply:
https://github.com/assertj/assertj-core/blob/c0f48d3858e047ed0421a36c435e6f1b5503edcb/src/main/java/org/assertj/core/api/AbstractDoubleArrayAssert.java#L835

It's already there